tests/main/views/test_users.py

```diff
@@ -336,61 +322,48 @@ def test_can_not_post_an_admin_ccs_user(self):
                     'name': 'joe bloggs'}}),
             content_type='application/json')
 
-        assert_equal(response.status_code, 400)
+        assert response.status_code == 400
         error = json.loads(response.get_data())['error']
-        assert_in("'admin-ccs' is not one of", error)
-
-    def test_can_post_a_supplier_user(self):
-        with self.app.app_context():
-            db.session.add(
-                Supplier(supplier_id=1, name=u"Supplier 1")
-            )
-            db.session.commit()
+        assert "'admin-ccs' is not one of" in error
 
+    def test_can_post_a_supplier_user(self, supplier_basic):
```

Comment:
Where does `supplier_basic` come from? It looks like it's just an ID?  In which case we could call it that?